### PR TITLE
*: add more contents about supported versions and upgrade methods

### PR DESCRIPTION
**Pull request by zimulala**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
 https://github.com/pingcap/tidb/issues/45885

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @Benjamin2037 @Frank945946

File: smooth-upgrade-tidb.md [hunk 1]
```diff
@@ -22,12 +27,27 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
 
 引入平滑升级后，TiDB 升级过程不再受上述限制。
 
-升级过程中，TiDB 会自动进行以下操作，无需用户干预：
+### 升级步骤
+
+#### TiUP 方式
+
+目前 TiUP v1.13.0 版本不支持此功能，下一个版本会自适应支持此功能(TiDB v7.5.0 时会发布此版本)，即不需要改动 `tiup cluster upgrade` 操作流程。
```
Comment (hfxsd) [inline smooth-upgrade-tidb.md:34]:
```suggestion
目前 TiUP v1.13.0 版本不支持此功能，下一个版本会自适应支持此功能 （TiDB v7.5.0 时会发布此版本），即不需要改动 `tiup cluster upgrade` 操作流程。
```

File: smooth-upgrade-tidb.md [hunk 2]
```diff
@@ -22,12 +27,27 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
...
 
-1. 暂停用户的 DDL 操作。
-2. 执行升级过程中的系统 DDL 操作。
-3. 恢复被暂停的用户的 DDL 操作。
-4. 完成升级。
+目前不支持此功能，会尽早自适应支持此功能。
+
+#### 其它方式
+
+如果手动或者自己脚本升级，需要如下操作：
```
Comment (hfxsd) [inline smooth-upgrade-tidb.md:42]:
```suggestion
如果手动升级或者使用脚本升级，需要如下操作：
```

File: smooth-upgrade-tidb.md [hunk 3]
```diff
@@ -22,12 +27,27 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
...
+
+1. 用户需要给集群中的任意一台 TiDB 发送 HTTP 升级开始请求：`curl -X POST http://{TiDBIP}:10080/upgrade/start`。
+   * TiDB 集群会进入 upgrading 状态。
+   * 接下来将要执行的用户的 DDL 操作都会被暂停。
+2. 用户替换 TiDB binary，并进行滚动升级。此过程和原升级过程一致。
+   * 执行升级过程中的系统 DDL 操作。
+3. 等集群中所有 TiDB 升级成功后，给任意一台 TiDB 发送 HTTP 升级结束请求：`curl -X POST http://{TiDBIP}:10080/upgrade/finsh`。
+   * 恢复被暂停的用户的 DDL 操作。
```
Comment (hfxsd) [inline smooth-upgrade-tidb.md:50]:
```suggestion
1. 你需要给集群中的任意一台 TiDB 发送 HTTP 升级开始请求：`curl -X POST http://{TiDBIP}:10080/upgrade/start`。
   * TiDB 集群会进入 **Upgrading** 状态。
   * 接下来将要执行的 DDL 操作都会被暂停。
2. 替换 TiDB binary，并进行滚动升级。此过程和原升级过程一致。
   * 执行升级过程中的系统 DDL 操作。
3. 等集群中所有 TiDB 升级成功后，给任意一台 TiDB 发送 HTTP 升级结束请求：`curl -X POST http://{TiDBIP}:10080/upgrade/finsh`。
   * 恢复被暂停的用户的 DDL 操作。
```

File: smooth-upgrade-tidb.md [hunk 4]
```diff
@@ -11,7 +11,12 @@ summary: 本文介绍支持无需手动取消 DDL 的平滑升级集群功能。
 
 本文档介绍 TiDB 的平滑升级集群功能，支持无需手动取消 DDL 的操作。
 
-从 v7.1.0 起，当将 TiDB 升级至更高的版本时，TiDB 支持平滑升级功能，取消了升级过程中的限制，提供更平滑的升级体验。此功能默认开启，且无开关控制。
+从 v7.1.0 起，当将 TiDB 升级至更高的版本时，TiDB 支持平滑升级功能，取消了升级过程中的限制，提供更平滑的升级体验。
```
Comment (Frank945946) [inline smooth-upgrade-tidb.md:14]: 是否可以再次强调一下是“升级前无需手动取消 DDL 的操作” 的这个限制？

File: smooth-upgrade-tidb.md [hunk 5]
```diff
@@ -11,7 +11,12 @@ summary: 本文介绍支持无需手动取消 DDL 的平滑升级集群功能。
...
+
+## 版本支持情况
+
+* 从 v7.1.0 升级到 [v7.1.1, v7.4.0）版本时，此功能默认开启，且无开关控制。
+* 从 v7.4.0 升级到更高的版本时, 通过是否发送 `/upgrade/start` HTTP 请求控制此功能开关。如果发送 `/upgrade/start` 请求，则开启此功能。具体方式可以参考：[TiDB HTTP API 文档](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)。
```
Comment (Frank945946) [inline smooth-upgrade-tidb.md:19]: 也就是 7.4.0 升级到更高版本，默认行为还是需要手动取消 DDL ？如果是，建议加一下备注。

**Comment by zimulala** [comment]
PTAL @Frank945946

File: smooth-upgrade-tidb.md [hunk 6]
```diff
@@ -56,3 +76,7 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
 * BR：BR 可能会将处于 paused 状态的 DDL 拷贝到 TiDB 中，而此状态的 DDL 不能自动 resume，可能导致后续 DDL 卡住的情况。
 
 * DM 和 TiCDC：如果在升级过程中使用 DM 和 TiCDC 向 TiDB 导入 SQL，并且其中包含 DDL 操作，则该导入操作会被阻塞，并可能出现未定义错误。
+
+### 插件使用限制
+
+TiDB 安装的插件可能自带 DDL 操作。在升级过程中，插件自带 DDL 操作中不能是非系统表的 DDL 操作，否则可能导致升级卡住情况。
```
Comment (Benjamin2037) [inline smooth-upgrade-tidb.md:82]: 插件带非系统表 DDL 操作有没有建议方法？

Comment (zimulala) [reply smooth-upgrade-tidb.md:82]: 没有好的办法，可以考虑将插件里面的 DDL 建于系统库上，或者不使用此功能

Comment (zimulala) [reply smooth-upgrade-tidb.md:82]: PTAL @Benjamin2037

File: smooth-upgrade-tidb.md [hunk 7]
```diff
@@ -11,7 +11,18 @@ summary: 本文介绍支持无需手动取消 DDL 的平滑升级集群功能。
...
+## 版本支持情况
+
+依据是否需要开关控制，具体可分为两种支持方式：
+* 无需开关控制，默认开启此功能的方式。目前支持此方式的版本分别是 v7.1.0，v7.1.1，v7.2.0，和 v7.3.0。具体支持升级版本的情况：
+  * 从 v7.1.0 升级到 v7.1.1 或 v7.2.0 或 v7.3.0 版本时。
+  * 从 v7.1.1 升级到 v7.2.0 或 v7.3.0 版本时。
+  * 从 v7.2.0 升级到 v7.3.0 版本时。
+* 通过是否发送 `/upgrade/start` HTTP 请求控制此功能开关。即此功能默认关闭，可通过发送 `/upgrade/start` 请求，开启此功能。具体方式可以参考：[TiDB HTTP API 文档](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)。具体版本情况：
+  * 从 v7.1.2 以及它之后的 v7.1 版本（即 v7.1.x，且 x >= 2）升级到 v7.4.0 及更高版本时。
+  * 从 v7.4.0 升级到更高的版本时。
```
Comment (hfxsd) [inline smooth-upgrade-tidb.md:25]:
```suggestion
依据是否需要开关控制，可分为两种支持方式：

* 无需开关控制，默认开启此功能的方式。目前支持此方式的版本分别是 v7.1.0，v7.1.1，v7.2.0，和 v7.3.0。具体支持升级版本的情况：
  * 从 v7.1.0 升级到 v7.1.1、v7.2.0 或 v7.3.0 版本
  * 从 v7.1.1 升级到 v7.2.0 或 v7.3.0 版本
  * 从 v7.2.0 升级到 v7.3.0 版本

* 通过是否发送 `/upgrade/start` HTTP 请求控制此功能开关。即此功能默认关闭，可通过发送 `/upgrade/start` 请求，开启此功能。具体方式可以参考：[TiDB HTTP API 文档](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)。具体版本情况：
  * 从 v7.1.2 以及它之后的 v7.1 版本（即 v7.1.x，且 x >= 2）升级到 v7.4.0 及更高版本时。
  * 从 v7.4.0 升级到更高的版本时。
```

File: smooth-upgrade-tidb.md [hunk 8]
```diff
@@ -20,14 +31,29 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
...
+
+如果手动升级或者使用脚本升级，需要如下操作：
+
+1. 你需要给集群中的任意一台 TiDB 发送 HTTP 升级开始请求：`curl -X POST http://{TiDBIP}:10080/upgrade/start`。
+   * TiDB 集群会进入 **Upgrading** 状态。
+   * 接下来将要执行的 DDL 操作都会被暂停。
+2. 替换 TiDB binary，并进行滚动升级。此过程和原升级过程一致。
+   * 执行升级过程中的系统 DDL 操作。
+3. 等集群中所有 TiDB 升级成功后，给任意一台 TiDB 发送 HTTP 升级结束请求：`curl -X POST http://{TiDBIP}:10080/upgrade/finsh`。
+   * 恢复被暂停的用户的 DDL 操作。
```
Comment (hfxsd) [inline smooth-upgrade-tidb.md:56]:
```suggestion
手动升级或者使用脚本升级的操作如下：

1. 给集群中的任意一台 TiDB 发送 HTTP 升级开始请求：`curl -X POST http://{TiDBIP}:10080/upgrade/start`。
   * TiDB 集群会进入 **Upgrading** 状态。
   * 接下来将要执行的 DDL 操作都会被暂停。

2. 替换 TiDB binary，并进行滚动升级。此过程和原升级过程一致。
   * 执行升级过程中的系统 DDL 操作。

3. 等集群中所有 TiDB 升级成功后，给任意一台 TiDB 发送 HTTP 升级结束请求：`curl -X POST http://{TiDBIP}:10080/upgrade/finsh`。
   * 恢复被暂停的用户的 DDL 操作。
```

**Comment by zimulala** [comment]
PTAL @Benjamin2037 @Frank945946

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Comment by zimulala** [comment]
PTAL @hfxsd

File: smooth-upgrade-tidb.md [hunk 9]
```diff
@@ -11,7 +11,20 @@ summary: 本文介绍支持无需手动取消 DDL 的平滑升级集群功能。
...
+依据是否需要开关控制，可分为两种支持方式：
+
+* 无需开关控制，默认开启此功能的方式。目前支持此方式的版本分别是 v7.1.0，v7.1.1，v7.2.0，和 v7.3.0。具体支持升级版本的情况：
+  * 从 v7.1.0 升级到 v7.1.1、v7.2.0 或 v7.3.0 版本
+  * 从 v7.1.1 升级到 v7.2.0 或 v7.3.0 版本
+  * 从 v7.2.0 升级到 v7.3.0 版本
+
+* 通过是否发送 `/upgrade/start` HTTP 请求控制此功能开关。即此功能默认关闭，可通过发送 `/upgrade/start` 请求，开启此功能。具体方式可以参考：[TiDB HTTP API 文档](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)。具体版本情况：
+  * 从 v7.1.2 以及它之后的 v7.1 版本（即 v7.1.x，且 x >= 2）升级到 v7.4.0 及更高版本
+  * 从 v7.4.0 升级到更高的版本
```
Comment (hfxsd) [inline smooth-upgrade-tidb.md:27]:
```suggestion
* 无需开关控制，默认开启此功能的方式。目前支持此方式的版本分别是 v7.1.0，v7.1.1，v7.2.0，和 v7.3.0。具体支持升级版本的情况：
    * 从 v7.1.0 升级到 v7.1.1、v7.2.0 或 v7.3.0 版本
    * 从 v7.1.1 升级到 v7.2.0 或 v7.3.0 版本
    * 从 v7.2.0 升级到 v7.3.0 版本

* 通过是否发送 `/upgrade/start` HTTP 请求控制此功能开关。即此功能默认关闭，可通过发送 `/upgrade/start` 请求，开启此功能。具体方式可以参考：[TiDB HTTP API 文档](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)。具体版本情况：
    * 从 v7.1.2 以及它之后的 v7.1 版本（即 v7.1.x，且 x >= 2）升级到 v7.4.0 及更高版本
    * 从 v7.4.0 升级到更高的版本
```

**Comment by zimulala** [comment]
PTAL @hfxsd

File: smooth-upgrade-tidb.md [hunk 10]
```diff
@@ -20,14 +33,31 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
...
+上述限制可概括为用户需要保证在升级过程中无用户 DDL 操作。引入平滑升级后，TiDB 升级过程不再受此限制。
+
+### 升级步骤
+
+#### TiUP 方式
+
+TiUP 会在 v1.14.0 版本自适应支持此功能，即无需特殊操作，直接使用 `tiup cluster upgrade` 操作流程即可。注意：目前不支持 `tiup cluster patch` 方式。
 
-升级过程中，TiDB 会自动进行以下操作，无需用户干预：
+#### Operator 方式
```
Comment (ran-huang) [inline smooth-upgrade-tidb.md:44]:
```suggestion
#### TiDB Operator 方式
```

File: smooth-upgrade-tidb.md [hunk 11]
```diff
@@ -20,14 +33,31 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
...
+手动升级或者使用脚本升级的操作如下：
+
+1. 给集群中的任意一台 TiDB 发送 HTTP 升级开始请求：`curl -X POST http://{TiDBIP}:10080/upgrade/start`。
+   * TiDB 集群会进入 **Upgrading** 状态。
+   * 接下来将要执行的 DDL 操作都会被暂停。
+
+2. 替换 TiDB binary，并进行滚动升级。此过程和原升级过程一致。
+   * 执行升级过程中的系统 DDL 操作。
+
+3. 等集群中所有 TiDB 升级成功后，给任意一台 TiDB 发送 HTTP 升级结束请求：`curl -X POST http://{TiDBIP}:10080/upgrade/finsh`。
```
Comment (ran-huang) [inline smooth-upgrade-tidb.md:59]:
```suggestion
3. 等集群中所有 TiDB 升级成功后，给任意一台 TiDB 发送 HTTP 升级结束请求：`curl -X POST http://{TiDBIP}:10080/upgrade/finish`。
```

File: smooth-upgrade-tidb.md [hunk 12]
```diff
@@ -56,3 +86,7 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
 * BR：BR 可能会将处于 paused 状态的 DDL 拷贝到 TiDB 中，而此状态的 DDL 不能自动 resume，可能导致后续 DDL 卡住的情况。
 
 * DM 和 TiCDC：如果在升级过程中使用 DM 和 TiCDC 向 TiDB 导入 SQL，并且其中包含 DDL 操作，则该导入操作会被阻塞，并可能出现未定义错误。
+
+### 插件使用限制
+
+TiDB 安装的插件可能自带 DDL 操作。在升级过程中，插件自带 DDL 操作中不能是非系统表的 DDL 操作，否则可能导致升级卡住情况。
```
Comment (ran-huang) [inline smooth-upgrade-tidb.md:92]: TiDB 安装的插件可能自带 DDL 操作。然而，在升级过程中，如果这些插件自带的 DDL 操作针对非系统表进行，可能导致升级过程出现问题。

File: smooth-upgrade-tidb.md [hunk 13]
```diff
@@ -11,7 +11,20 @@ summary: 本文介绍支持无需手动取消 DDL 的平滑升级集群功能。
...
+从 v7.1.0 起，当将 TiDB 升级至更高的版本时，TiDB 支持平滑升级功能，取消了升级过程中的限制（用户需要保证升级过程中无用户 DDL 操作），提供更平滑的升级体验。
+
+## 版本支持情况
+
+依据是否需要开关控制，可分为两种支持方式：
+
+* 无需开关控制，默认开启此功能的方式。目前支持此方式的版本分别是 v7.1.0，v7.1.1，v7.2.0，和 v7.3.0。具体支持升级版本的情况：
+    * 从 v7.1.0 升级到 v7.1.1、v7.2.0 或 v7.3.0 版本
+    * 从 v7.1.1 升级到 v7.2.0 或 v7.3.0 版本
+    * 从 v7.2.0 升级到 v7.3.0 版本
```
Comment (ran-huang) [inline smooth-upgrade-tidb.md:23]: 这里的 v7.3.0 是不是应该更新了？改成 v7.4.0 或者 v7.5.0？

Comment (zimulala) [reply smooth-upgrade-tidb.md:23]: 不用，这里就是 v7.3.0。v7.4.0 和更高版本是需要通过下面这种方式升级的

File: smooth-upgrade-tidb.md (hunk 12 above)
Comment (zimulala) [inline smooth-upgrade-tidb.md:92]:
```suggestion
TiDB 安装的插件可能自带 DDL 操作。然而，在升级过程中，如果这些插件自带的 DDL 操作针对非系统表进行，可能导致升级过程出现问题。
```

**Comment by zimulala** [comment]
PTAL
@ran-huang

File: smooth-upgrade-tidb.md (hunk 13 above)
Comment (ran-huang) [reply smooth-upgrade-tidb.md:23]: 7.3 升级到 7.4 属于哪一种情况

File: smooth-upgrade-tidb.md [hunk 14]
```diff
@@ -11,7 +11,20 @@ summary: 本文介绍支持无需手动取消 DDL 的平滑升级集群功能。
 
 本文档介绍 TiDB 的平滑升级集群功能，支持无需手动取消 DDL 的操作。
 
-从 v7.1.0 起，当将 TiDB 升级至更高的版本时，TiDB 支持平滑升级功能，取消了升级过程中的限制，提供更平滑的升级体验。此功能默认开启，且无开关控制。
+从 v7.1.0 起，当将 TiDB 升级至更高的版本时，TiDB 支持平滑升级功能，取消了升级过程中的限制（用户需要保证升级过程中无用户 DDL 操作），提供更平滑的升级体验。
```
Comment (hfxsd) [inline smooth-upgrade-tidb.md:14]:
```suggestion
从 v7.1.0 起，当将 TiDB 升级至更高的版本时，TiDB 支持平滑升级功能，取消了升级过程中的限制（你需要保证升级过程中无 DDL 操作），提供更平滑的升级体验。
```

File: smooth-upgrade-tidb.md [hunk 15]
```diff
@@ -20,14 +33,31 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
 - 在升级过程中执行 DDL 操作，TiDB 可能会出现未定义的行为。
 - 在 DDL 操作执行过程中升级 TiDB，TiDB 可能会出现未定义的行为。
 
-引入平滑升级后，TiDB 升级过程不再受上述限制。
+上述限制可概括为用户需要保证在升级过程中无用户 DDL 操作。引入平滑升级后，TiDB 升级过程不再受此限制。
```
Comment (hfxsd) [inline smooth-upgrade-tidb.md:36]:
```suggestion
上述限制可概括为，你需要保证在升级过程中无用户发起的 DDL 操作。引入平滑升级后，TiDB 升级过程不再受此限制。
```

File: smooth-upgrade-tidb.md [hunk 16]
```diff
@@ -20,14 +33,31 @@ TiDB 引入平滑升级功能前，对于升级过程中的 DDL 操作有如下
...
+
+### 升级步骤
+
+#### TiUP 方式
+
+TiUP 会在 v1.14.0 版本自适应支持此功能，即无需特殊操作，直接使用 `tiup cluster upgrade` 操作流程即可。注意：目前不支持 `tiup cluster patch` 方式。
```
Comment (hfxsd) [inline smooth-upgrade-tidb.md:42]:
```suggestion
TiUP 会在 v1.14.0 版本自适应支持此功能，即无需特殊操作，直接使用 `tiup cluster upgrade` 操作流程即可。注意目前不支持 `tiup cluster patch` 方式。
```

File: smooth-upgrade-tidb.md (hunk 13 above)
Comment (zimulala) [reply smooth-upgrade-tidb.md:23]: 都不算，不支持，这里只写了支持的版本

**Review by ran-huang — APPROVED** [review]

File: smooth-upgrade-tidb.md (hunk 13 above)
Comment (ran-huang) [reply smooth-upgrade-tidb.md:23]: 不支持的情况也说明一下吧

**Comment by zimulala** [comment]
PTAL @hfxsd @tangenta

File: smooth-upgrade-tidb.md [hunk 17]
```diff
@@ -11,7 +11,20 @@ summary: 本文介绍支持无需手动取消 DDL 的平滑升级集群功能。
...
+
+依据是否需要开关控制，可分为两种支持方式：
+
+* 无需开关控制，默认开启此功能的方式。目前支持此方式的版本分别是 v7.1.0，v7.1.1，v7.2.0，和 v7.3.0。具体支持升级版本的情况：
+    * 从 v7.1.0 升级到 v7.1.1、v7.2.0 或 v7.3.0 版本
+    * 从 v7.1.1 升级到 v7.2.0 或 v7.3.0 版本
+    * 从 v7.2.0 升级到 v7.3.0 版本
+
+* 通过是否发送 `/upgrade/start` HTTP 请求控制此功能开关。即此功能默认关闭，可通过发送 `/upgrade/start` 请求，开启此功能。具体方式可以参考：[TiDB HTTP API 文档](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)。具体版本情况：
+    * 从 v7.1.2 以及它之后的 v7.1 版本（即 v7.1.x，且 x >= 2）升级到 v7.4.0 及更高版本
```
Comment (zimulala) [inline smooth-upgrade-tidb.md:26]:
```suggestion
    * 从 v7.1.2 以及它之后的 v7.1 版本（即 >= v7.1.2）升级到 v7.4.0 及更高版本
```

**Review by tangenta — APPROVED** [review]

File: smooth-upgrade-tidb.md [hunk 18]
```diff
@@ -5,13 +5,34 @@ summary: 本文介绍支持无需手动取消 DDL 的平滑升级集群功能。
...
+| v7.1.1                               	 | v7.2.0 或 v7.3.0         	| 无需额外操作，自动支持平滑升级                                                                                                                                	| 实验特性                                                                      	|
+| v7.2.0                               	 | v7.3.0                   	| 无需额外操作，自动支持平滑升级                                                                                                                                	| 实验特性                                                                      	|
+| [v7.1.2, v7.2.0)                     	 | [v7.1.2, v7.2.0)         	| 通过发送 `/upgrade/start` HTTP 请求开启平滑升级，具体方式有两种：[TiUP 方式](/smooth-upgrade-tidb.md#TiUP 方式)；[其它方式](/smooth-upgrade-tidb.md#其它方式) 	| 不开启平滑升级时，需自行保证升级时无 DDL 操作。                               	|
+| [v7.1.2, v7.2.0) 或 >= v7.4.0        	  | >= v7.4.0                	| 通过发送 `/upgrade/start` HTTP 请求开启平滑升级，具体方式有两种：[TiUP 方式](/smooth-upgrade-tidb.md#TiUP 方式)；[其它方式](/smooth-upgrade-tidb.md#其它方式) 	| 不开启平滑升级时，需自行保证升级时无 DDL 操作。                               	|
```
Comment (zimulala) [inline smooth-upgrade-tidb.md:34]:
```suggestion
| [v7.1.2, v7.2.0)                     	 | [v7.1.2, v7.2.0)         	| 通过发送 `/upgrade/start` HTTP 请求开启平滑升级，具体方式有两种：[TiUP 方式](/smooth-upgrade-tidb.md#tiup-方式))；[其它方式](/smooth-upgrade-tidb.md#其它方式) 	| 不开启平滑升级时，需自行保证升级时无 DDL 操作。                               	|
| [v7.1.2, v7.2.0) 或 >= v7.4.0        	  | >= v7.4.0                	| 通过发送 `/upgrade/start` HTTP 请求开启平滑升级，具体方式有两种：[TiUP 方式](/smooth-upgrade-tidb.md#tiup-方式))；[其它方式](/smooth-upgrade-tidb.md#其它方式) 	| 不开启平滑升级时，需自行保证升级时无 DDL 操作。                               	|
```

**Review by ran-huang — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**